### PR TITLE
Various minor cleanups

### PR DESCRIPTION
**Pull request by smartcontracts**
## Description

Ported over some changes from my branch `polish/standards`, attempting to break into smaller PRs. This is entirely cleanup and does not change any functionality.

## Contributing Agreement

- [x] I have read and understood the [Optimism Contributing Guide and Code of Conduct](https://github.com/ethereum-optimism/optimism-monorepo/blob/master/.github/CONTRIBUTING.md) and am following those guidelines in this pull request.

### CODE REVIEW

**Comment by ben-chain** [comment]
@maurelian has been doing some cleanup as well, I will defer to him on these.  Hopefully no notable conflicts with what he's been working on :)

File: contracts/optimistic-ethereum/OVM/precompiles/OVM_L2ToL1MessagePasser.sol [hunk 1]
```diff
@@ -32,9 +34,6 @@ contract OVM_L2ToL1MessagePasser is iOVM_L2ToL1MessagePasser {
         override
         public
     {
-        // Note: although this function is public, only messages sent from the OVM_L2CrossDomainMessenger
```
Comment (maurelian) [inline contracts/optimistic-ethereum/OVM/precompiles/OVM_L2ToL1MessagePasser.sol:35]: Why remove this comment?

File: contracts/optimistic-ethereum/iOVM/accounts/iOVM_ECDSAContractAccount.sol [hunk 2]
```diff
@@ -14,11 +14,26 @@ interface iOVM_ECDSAContractAccount {
...
+    /**
+     * Executes a signed transaction.
+     * @param _transaction Signed EOA transaction.
+     * @param _signatureType Hashing scheme used for the transaction (e.g., ETH signed message).
+     * @param _v Signature `v` parameter.
+     * @param _r Signature `r` parameter.
+     * @param _s Signature `s` parameter.
+     * @return Whether or not the call returned (rather than reverted).
+     * @return Data returned by the call.
+     */
```
Comment (maurelian) [inline contracts/optimistic-ethereum/iOVM/accounts/iOVM_ECDSAContractAccount.sol:26]: The same comment is in the implementation contract. Should we strive to have these comments in both places?

**Review by maurelian — COMMENTED** [review]
I like the approach to whitespace, and removing the return variable names. 

Just a couple questions left on the diff.

**Pull request closed without merging**